packages/components/tests/dummy/app/templates/components/form/file-input.hbs

```diff
@@ -0,0 +1,80 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Form::FileInput Component"}}
+
+<Shw::Text::H1>Form::FileInput</Shw::Text::H1>
+
+<section data-test-percy>
+  <button id="dummy-toggle-highlight" type="button" {{on "click" this.toggleHighlight}}>
+    {{if this.showHighlight "Hide" "Show"}}
+    layout highlight
+  </button>
+
+  <div class="{{if this.showHighlight 'shw-component-form-file-input-layout-highlight'}}">
+    <Shw::Text::H2>On its own</Shw::Text::H2>
+    <Shw::Text::Body>
+      <code>Input[type=file]</code>
+      (styled via
+      <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/::file-selector-button">
+        <code>::file-selector-button</code>
+      </a>)
+    </Shw::Text::Body>
+    <Hds::Form::FileInput />
+
+    <Shw::Text::H2>Used next to other content</Shw::Text::H2>
```

Comment:
I think we can group these examples with the first one under a 'Content' section – as you did for the [Reveal](https://hds-showcase-git-hds-2278-file-upload-component-hashicorp.vercel.app/components/reveal).

Reply:
The Showcase is still very much a WIP as the final designs aren't ready yet.

Reply:
I've cleaned up the Showcase more and added examples of base and field components.

Let me know if there are any other useful examples you think I should add.